src/include/duckdb/storage/checkpoint/rle_sort_options.hpp

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/storage/checkpoint/rle_sort_options.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/common/constants.hpp"
+
+namespace duckdb {
+
+enum class RLESortOption : uint8_t {
```

Comment:
Is this enum necessary, as we only have one option?

Reply:
I could remove it, but I added this to account for the possibility of more reordering options being added in the future. Let me know what you prefer!